presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -1098,16 +1098,38 @@ bool TaskManager::getStuckOpCalls(
     std::vector<velox::exec::Task::OpCallInfo>& stuckOpCalls) const {
   const auto thresholdDurationMs =
       SystemConfig::instance()->driverStuckOperatorThresholdMs();
+  const auto thresholdCancelMs =
```

Comment:
NYC: shall we call stuckOpCalls.clear() on function entrance? Or VELOX_CHECK that stuckOpCalls is empty? Thanks!